### PR TITLE
OCPQE-19089: automate OCP-69177

### PR DESCRIPTION
**Pull request by jinyunma**
OCP-69177 [ipi-on-azure] install cluster with mixed publish, external apiserver, external ingress

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-azure-ipi-publish-mixed-apiserver-ingress-external-f14

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-azure-ipi-mixed-apiserver-ingress-external-f14

**Comment by jinyunma** [comment]
tested job: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/47952/rehearse-47952-periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-azure-ipi-mixed-apiserver-ingress-external-f14/1749700077200347136

cc @jianlinliu @gpei to review

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/publish-mixed/apiserver-ingress-external/provision/cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - chain: ipi-conf-azure
+  - ref: ipi-conf-operator-publish-strategy
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-azure-publish-mixed
+  env:
+  - name: APISERVER_PUBLISH_STRATEGY
+    default: "External"
+  - name: INGRESS_PUBLISH_STRATEGY
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/publish-mixed/apiserver-ingress-external/provision/cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external-provision-chain.yaml:13]: If both are external, is that the same as `publish: external` without `operatorPublishingStrategy` setting?

Comment (gpei) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/publish-mixed/apiserver-ingress-external/provision/cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external-provision-chain.yaml:13]: yeah, it's true, I remember I discussed with Jinyun before on this, setting both as External is indeed allowed in this way so maybe we also need to test it.

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/publish-mixed/apiserver-ingress-external/provision/cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external-provision-chain.yaml:13]: yes, it equals to` publish: external`, just to validate that installation have no issue in such configuration.

**Pull request merged**